Bluejay/Bluejay/Connection.swift

```diff
@@ -121,15 +121,15 @@ class Connection: Queueable {
         updateQueue()
     }
     
-    func fail(_ error: NSError) {
+    func fail(_ error: Swift.Error) {
```

Comment:
Does the Error protocol need to be fully qualified here? Seems like just `Error` should work. There are a couple other places that it's sued fully qualified that I'm not sure if it needs to.

Reply:
Done